File: backend/src/database/mongoDB.js
const mongoose = require("mongoose");

const host = process.env.MONGO_HOST;
const port = process.env.MONGO_PORT;
const user = process.env.MONGO_USER;
const password = process.env.MONGO_PASSWORD;
const database = process.env.MONGO_NAME;

exports.conectar = async () => {
  await mongoose
    .connect(
      `mongodb://${user}:${password}@${host}:${port}/${database}?ssl=true`,
      { useNewUrlParser: true, useUnifiedTopology: true }
    )
    .then(() =>
      console.log("MongoDB is connected to:", mongoose.connection.name)
    )
    .catch((e) => console.log(e));
};

File: api/go/src/database/mongodb.go
package database

import (
	"os"

	"go.mongodb.org/mongo-driver/mongo/options"
)

func GetClient() *options.ClientOptions {
	user := os.Getenv("MONGO_USER")
	password := os.Getenv("MONGO_PASSWORD")
	host := os.Getenv("MONGO_HOST")
	port := os.Getenv("MONGO_PORT")
	database := os.Getenv("MONGO_NAME")
	client := options.Client().ApplyURI("mongodb://" + user + ":" + password + "@" + host + ":" + port + "/" + database + "?ssl=true&replicaSet=globaldb&retrywrites=false")
	return client
}
